index.html

```diff
@@ -1071,22 +933,24 @@
     </p>
     <ul>
       <li>
-        Only <a>secure context</a>s are allowed to access <a>NFC content</a>.
+        Only <a>secure contexts</a> are allowed to access <a>NFC content</a>.
         Browsers may ignore this rule for development purposes only.
       </li>
       <li>
-        In order to use NFC, the <code><a>Window</a></code> object associated
-        with the <code><a>Document</a></code> contained by the
+        In order to use NFC, the {{Window}} object associated
+        with the {{Document}} contained by the
         <a>top-level browsing context</a> using the Web NFC API must be
-        <a>visible</a> and in <a>focus</a>.
+        <a data-cite="page-visibility/#dom-visibilitystate-visible">visible</a>
+        and in <a data-cite="html#focus">focus</a>.
         This also means that UAs should block access to the NFC radio if
         the display is off or the device is locked.
         For backgrounded web pages, receiving and pushing <a>NFC content</a>
         must be <a id="#nfc-suspended">suspended</a>.
       </li>
       <li>
-        When pushing <a>Web NFC content</a>, the <a>serialized origin</a> and
-        the <a>URL path</a> of the <a>current settings object</a> when
+        When pushing <a>Web NFC content</a>, the
+        <a data-cite="html#ascii-serialisation-of-an-origin">serialized origin</a>
```

Comment:
Same here... find out why this was not exported from HTML if you need it. Maybe there is something in the URL spec you can use.

Reply:
I can find this in  xref UI, as it uses a different name(serialized origin), that's why we need a "data-cite" here, right?

Reply:
Nope :) it means there is a spec bug. If something hasn’t been exported from HTML, that warrants further investigation. Basically, we need to treat every data-cite as a bug 🕷 it should only ever be needed for specs that haven’t exported their terms.

Reply:
Oh wait, I totally misread. You are correct!

Reply:
But, don’t use “data-cite”, use data-lt="serialized origin"

Reply:
Thank you, tips learned.